Source/FORMDataSource.h

```diff
@@ -65,6 +65,7 @@ typedef UICollectionViewCell * (^FORMFieldConfigureCellForItemAtIndexPath)(FORMF
 - (NSInteger)numberOfFields;
 - (NSArray *)forms;
 - (NSDictionary *)valuesDictionary;
+- (NSDictionary *)removedDynamicValues;
```

Comment:
I thought about that but it's only the dynamic values that gets added to that property to it made sense to make it more explicit. From an API perspective it makes sense but `removedValues` is shorter and reads nice.